commands/invoke.go

```diff
@@ -71,7 +71,11 @@ func runInvoke(cmd *cobra.Command, args []string) {
 		gatewayAddress = gateway
 	}
 
-	fmt.Fprintf(os.Stderr, "Reading from STDIN - hit (Control + D) to stop.\n")
+	stat, _ := os.Stdin.Stat()
+	if (stat.Mode() & os.ModeCharDevice) != 0 {
+		fmt.Fprintf(os.Stderr, "Reading from STDIN - hit (Control + D) to stop.\n")
```

Comment:
Why to Stderr, not Stdout? Won't this cause TravisCi to break?

Reply:
Nope, stderr won't impact travis - that responds to the exit code from the command.

You use stderr for informational messages that you don't want to impact the use of the command when chained together with other commands, for example piping the output from this to another command would only see the output written to stdout passed to the next command.